README.md

```diff
@@ -106,6 +106,7 @@ React element of the application.
 | **nextRoot**                   | Absolute path to Next.js root folder                                               | `string`                          | _auto detected_ |
 | **dotenvFile**                 | Relative path to a `.env` file holding [environment variables][next-docs-env-vars] | `string`                          | -               |
 | **wrapper**                    | Map of render functions. Useful to decorate component tree with mocked providers.  | `{ Page?: NextPage => NextPage }` | -               |
+| **nonIsolatedModules**         | List of modules that should preserve identity between client and server context.   | `string[]`                        | []              |
```

Comment:
What if we removed the negation? Something like `preserveModuleIdentity`?

Same could apply for our internal methods like: `unIsolatePredefinedJestModules ` , `unIsolateJestModules `..

Reply:
Hmm. Yeah naming this is confusing.... What about something less abstract for users to understand: `sharedModules` -- since their identity is "shared" between `client` and `server`.

Reply:
Ok. Let's go with `sharedModules`!

Reply:
Done 👍